src/chainstate/burn/mod.rs

```diff
@@ -24,16 +24,17 @@ pub mod distribution;
 pub mod operations;
 pub mod sortition;
 
-pub const CHAINSTATE_VERSION: &'static str = "21.0.0.0";
+pub const CHAINSTATE_VERSION: &'static str = "22.0.0.0";
```

Comment:
Should this be defined in `core/mod.rs` alongside the byte array version string?

Reply:
Sure, I can move it there.